### PR TITLE
PLA 1534 signing request

### PR DESCRIPTION
**Pull request by coolguru**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ make build
...
+  coreo [command]
+
+Available Commands:
+  configure   create a new configuration
+  me          gets me
+
+Flags:
+      --config string    config file (default is $HOME/.cloudcoreo/profiles.yaml)
+      --profile string   user profile (default) (default "default")
+  -t, --toggle           Help message for toggle
```
Comment (pauldallen) [inline README.md]: what is toggle?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ make build
...
+Usage:
+  coreo [command]
+
+Available Commands:
+  configure   create a new configuration
+  me          gets me
+
+Flags:
+      --config string    config file (default is $HOME/.cloudcoreo/profiles.yaml)
+      --profile string   user profile (default) (default "default")
```
Comment (pauldallen) [inline README.md]: lets add '-p' too

Comment (coolguru) [inline README.md]: any global flag doesnt support short hand, same is true for aws as well.

File: README.md (hunk 1 above)
Comment (coolguru) [inline README.md]: Im not sure, it was part of the original code. We can remove it.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,26 @@ make build
 ```
 
 ###  Commands
-'''
+```
+A longer description that spans multiple lines and likely contains
+examples and usage of using your application. For example:
+
+Cobra is a CLI library for Go that empowers applications.
```
Comment (coolguru) [inline README.md]: Ok, will remove it in next PR.

**Pull request merged**